core/src/main/scala/kafka/controller/PartitionStateMachine.scala

```diff
@@ -450,42 +449,36 @@ class ZkPartitionStateMachine(config: KafkaConfig,
    *
    * @param leaderIsrAndControllerEpochs set of partition to determine if unclean leader election should be
    *                                     allowed
-   * @param allowUnclean whether to allow unclean election without having to read the topic configuration
+   * @param forceUncleanElection whether to force unclean election without having to read the topic configuration
    * @return a sequence of three element tuple:
    *         1. topic partition
-   *         2. leader, isr and controller epoc. Some means election should be performed
+   *         2. leader, isr and controller epoch. Some means election should be performed
    *         3. allow unclean
    */
-  private def collectUncleanLeaderElectionState(
-    leaderAndIsrs: Seq[(TopicPartition, LeaderAndIsr)],
-    allowUnclean: Boolean
+  private def collectUncleanLeaderElectionState(leaderAndIsrs: Seq[(TopicPartition, LeaderAndIsr)],
+    forceUncleanElection: Boolean
   ): Seq[(TopicPartition, Option[LeaderAndIsr], Boolean)] = {
     val (partitionsWithNoLiveInSyncReplicas, partitionsWithLiveInSyncReplicas) = leaderAndIsrs.partition {
       case (partition, leaderAndIsr) =>
         val liveInSyncReplicas = leaderAndIsr.isr.filter(controllerContext.isReplicaOnline(_, partition))
         liveInSyncReplicas.isEmpty
     }
 
-    val electionForPartitionWithoutLiveReplicas = if (allowUnclean) {
+    val electionForPartitionWithoutLiveReplicas = if (forceUncleanElection) {
```

Comment:
nit: while we're at it, should this be `electionForPartition` => `electionForPartitions` ?